internal/api/router/manager.go

```diff
@@ -27,18 +27,20 @@ type DevicesRepository interface {
 }
 
 type Manager struct {
-	devices DevicesRepository
-	reports *report.Repository
+	devices        DevicesRepository
+	reports        *report.Repository
+	restartRequest chan<- struct{}
```

Comment:
maybe rebuildRequest is a better name ?

Reply:
changes to newBuildRequest for consistency